src/recommendationservice/requirements.txt

```diff
@@ -1,13 +1,8 @@
 google-api-core==2.4.0
 grpcio-health-checking==1.43.0
 grpcio==1.43.0
-opentelemetry-api==1.13.0
-opentelemetry-exporter-otlp-proto-grpc==1.13.0
-opentelemetry-instrumentation==0.34b0
-opentelemetry-instrumentation-grpc==0.34b0
-opentelemetry-instrumentation-urllib3==0.34b0
-opentelemetry-sdk==1.13.0
-python-dotenv==0.20.0
-python-json-logger==2.0.2
-requests==2.27.1
-urllib3==1.26.8
\ No newline at end of file
+opentelemetry-distro==0.34b0
+opentelemetry-exporter-otlp==1.13.0
```

Comment:
I'd continue using the previous lib:
`opentelemetry-exporter-otlp-proto-grpc`
just for clarity because we are not planning to export http.